atomic_reactor/plugins/pre_fetch_sources.py

```diff
@@ -135,7 +135,8 @@ def download_sources(self, sources, insecure=False, download_dir=SRPMS_DOWNLOAD_
         :param download_dir: str, directory where to download content
         :return: str, paths to directory with downloaded sources
         """
-        workdir = tempfile.mkdtemp()
+        os.makedirs(os.path.join(self.workflow.source.workdir, 'temp'), exist_ok=True)
+        workdir = tempfile.mkdtemp(dir=os.path.join(self.workflow.source.workdir, 'temp'))
```

Comment:
how about `{workdir}/inputs/` or `{workdir}/sources/` instead?

Reply:
updated to `{workdir}/sources/`